### PR TITLE
FAT-5021/C353980

### PR DESCRIPTION
**Pull request by re-jas**
- [C353980: Check that user can choose the item to which to move Title Level Request](https://issues.folio.org/browse/FAT-5021)
- [Allure](https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/2411/allure/)

### CODE REVIEW

File: cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        });
+        cy.getMaterialTypes({ limit: 1 }).then((materialTypes) => {
+          testData.materialTypeId = materialTypes.id;
+        });
+      })
+      .then(() => {
+        instanceData.itemsData = [
+          {
+            barcode: generateUniqueItemBarcodeWithShift(),
+            status: { name: 'Available' },
```
Comment (penfriend) [inline cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js:61]: please add available item status and add it to the constants file: cypress\support\constants.js

**Review by Sherzod-Kenjaev — APPROVED** [review]

File: cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            permanentLoanType: { id: testData.loanTypeId },
+            materialType: { id: testData.materialTypeId },
+          },
+          {
+            barcode: generateUniqueItemBarcodeWithShift(),
+            status: { name: 'Available' },
```
Comment (penfriend) [inline cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js:67]: the same here.

File: cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+            permanentLoanType: { id: testData.loanTypeId },
+            materialType: { id: testData.materialTypeId },
+          },
+          {
+            barcode: generateUniqueItemBarcodeWithShift(),
+            status: { name: 'Available' },
```
Comment (penfriend) [inline cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js:73]: please add available item status and add it to the constants file: cypress\support\constants.js

File: cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+        UserEdit.addServicePointViaApi(
+          testData.userServicePoint.id,
+          userData.userId,
+          testData.userServicePoint.id
+        );
+      })
+      .then(() => {
+        TitleLevelRequests.changeTitleLevelRequestsStatus('allow');
+        Requests.createNewRequestViaApi({
+          fulfilmentPreference: 'Hold Shelf',
```
Comment (penfriend) [inline cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js:113]: please add fulfilmentPreference  to CY_ENV to the constants file: cypress\support\constants.js

File: cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+          instanceId: instanceData.instanceId,
+          pickupServicePointId: testData.userServicePoint.id,
+          requestDate: new Date(),
+          requestLevel: 'Title',
```
Comment (penfriend) [inline cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js:117]: please add title and item level requests and add it to the constants file: cypress\support\constants.js

File: cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+          requestType: 'Page',
```
Comment (penfriend) [inline cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js:118]: please add page type of requests and add it to the constants file: cypress\support\constants.js

File: cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js (hunk 1 above)
Comment (re-jas) [reply cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js:61]: done

File: cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js (hunk 2 above)
Comment (re-jas) [reply cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js:67]: done

File: cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js (hunk 3 above)
Comment (re-jas) [reply cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js:73]: done

File: cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js (hunk 6 above)
Comment (re-jas) [reply cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js:118]: done

File: cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js (hunk 4 above)
Comment (re-jas) [reply cypress/e2e/title-level-request/user-can-choose-item-to-move-tlr.cy.js]: done

**Pull request merged**